helm/charts/dsg-runner-infra/templates/sensor.yaml

```diff
@@ -31,7 +31,7 @@ spec:
                     restartPolicy: Never
                     containers:
                       - name: dsg-container
-                        image: "407256539111.dkr.ecr.us-east-1.amazonaws.com/amplication-data-service-generator-runner:sha-04dcb0f"
+                        image:  {{ .Values.job.image }}
```

Comment:
Can i suggest that we use two values 'calls' to fill the `image`. Since the `repository` where the image comes from is always the same but only the tag is different:

e.g. job
```
- name: dsg-container
   image: {{ .Values.job.image.repository }}:{{ .Values.job.image.tag }}
```

e.g. values file
```
job:
    image:
        repository: "407256539111.dkr.ecr.us-east-1.amazonaws.com/amplication-data-service-generator-runner"
        tag: "latest"
```

It would allow us to have a pipeline change only the tag when we're in a situation where the argocd image updater still doesnt work in conjunction with vault. Thoughts?

Reply:
By the way this is also the way it is done when you do a `helm create <name>`.

Reply:
It sound good.